.github/pull_request_template.md

```diff
@@ -1,26 +1,29 @@
-<!-- Mark items with [x] where applicable -->
-
-#### General
-- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)
+<!-- Uncomment relevant sections and delete the lines which are not applicable -->
+<!--
+## New package
+- This new package conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)
+- This new package doesn't conform to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements), but I believe an exception should be made because xxx
+-->
 
-#### Have the results of the proposed changes been tested?
-- [ ] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
-- [ ] I generally don't use the affected packages but briefly tested this PR
+## Testing the changes
+- I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
+- I generally don't use the affected packages but briefly tested this PR
 
 <!--
 If GitHub CI cannot be used to validate the build result (for example, if the
 build is likely to take several hours), make sure to
 [skip CI](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#continuous-integration).
-When skipping CI, uncomment and fill out the following section.
 Note: for builds that are likely to complete in less than 2 hours, it is not
 acceptable to skip CI.
+
+Ignore the following section if this PR doesn't skip CI.
```

Comment:
Whole section could be shortened to something like

> Build takes over 2h so I build locally for following archs, and am \[ci skipping]()

Reply:
I shortened it a bit, but it's still long.